### ISSUE TITLE
Bash not found in node-dev-vite docker image

### ISSUE THREAD

**Issue by dGilli** [nystudio107/craft#90]
### Describe the bug
Bash seems to be no longer found in image `node-dev-vite` as of a couple weeks ago. I cant seem to find any upstream changes in [https://github.com/nystudio107/docker-images/blob/develop/node-dev-base/node-16-alpine/Dockerfile](https://github.com/nystudio107/docker-images/blob/develop/node-dev-base/node-16-alpine/Dockerfile) or any related dependencies.

I tried to just docker run the `node:16-alpine` and call bash but it seems as if it has never even been part of the image. Anyone else experiencing the same issue?

Starting the project (`make dev`) will error out the vite continer: `tester-vite-1 exited with code 127`

### To reproduce
Steps to reproduce the behaviour:
1. Start project normally (`make dev`) or run any upstream image: `docker run --rm -it <PICK ONE: nystudio107/node-dev-base:16-alpine, node:16-alpine or alpine:3.18> bash`
2. Container errors

### Expected behaviour
Bash should be present and work as it always has been.

### Versions
- Craft version: 3.8.15
- Docker version: 4.21.1 (114176)
- OS: macOS 12.2

**Comment by dGilli** [comment]
### Quick fix for anyone that has the same issue:

In the end I worked around it by simply changing the shebang from `#!/bin/bash
` to `#!/bin/sh` in [/docker-config/node-dev-vite/npm_install.sh](https://github.com/nystudio107/craft/blob/craft-vite/docker-config/node-dev-vite/npm_install.sh#L1C1-L1C1). Which seems like a reasonable fix to me

**Comment by khalwat** [comment]
yep, this is the right fix. Ironically, I just found and fixed this myself today! :) 

Here's the fix... apparently `bash` was removed from the base image, and the `npm_install.sh` script has the shebang `#!/bin/bash`... so it just needs to be changed to `#!/bin/sh` and then do
```
docker-compose up --build
```
and away you go. ref: https://github.com/nystudio107/devmode/commit/d7560bad0f8dd1b58bae97eaeb3782738e5c8f71
